sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -607,15 +607,15 @@
     <bpmn:sequenceFlow id="Flow_1eulup9" sourceRef="Event_1sg75ph" targetRef="Activity_1wciqrl" />
     <bpmn:sequenceFlow id="Flow_1lihxee" sourceRef="Gateway_16zyh99" targetRef="Gateway_0rn7rsj" />
     <bpmn:sequenceFlow id="Flow_0jvb1k3" name="Exempt" sourceRef="Gateway_0rn7rsj" targetRef="Event_0sz9je4">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=studentDataIndigenousStatus = "yes" or studentPDStatus = true or calculatedDataTotalEligibleDependants &gt; 0 or studentDataYouthInCare = "yes"</bpmn:conditionExpression>
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=studentDataIndigenousStatus = "yes" or calculatedDataApplicationPDPPDStatus = true or calculatedDataTotalEligibleDependants &gt; 0 or studentDataYouthInCare = "yes"</bpmn:conditionExpression>
```

Comment:
Just to share with DEVs, the idea of having the `calculatedDataApplicationPDPPDStatus` instead of using the `applicationPDPPDStatus` is to keep it as a boolean (that is what is needed right now) and it would also allow further logic to be introduced when we will need to deal with the PD expiration in the near future.